README.md

```diff
@@ -4,7 +4,7 @@
 ![Lint](https://github.com/angristan/openvpn-install/workflows/Lint/badge.svg)
 ![visitors](https://visitor-badge.glitch.me/badge?page_id=angristan.openvpn-install)
 
-OpenVPN installer for Debian, Ubuntu, Fedora, CentOS and Arch Linux.
+OpenVPN installer for Debian, Ubuntu, Fedora, CentOS, Oracle Linux 8 and Arch Linux.
```

Comment:
```suggestion
OpenVPN installer for Debian, Ubuntu, Fedora, CentOS, Oracle Linux and Arch Linux.
```

Reply:
fixed in commit [05cf140ffba4c7f5f3811e5563f37bfd13f72373](https://github.com/angristan/openvpn-install/pull/810/commits/05cf140ffba4c7f5f3811e5563f37bfd13f72373)